frontend/src/components/Question/index.jsx

```diff
@@ -1,15 +1,27 @@
 import PropTypes from "prop-types";
 import Style from "./style";
 
-export default function Question({ data: { question, category } }) {
+export default function Question({
+  data: { question },
+  categoryName,
+  categoryColor,
```

Comment:
Are these informations not stored in the Context ? Passing them as props here is useless :(